scripts/ATVdetailsSender.sh

```diff
@@ -86,6 +87,24 @@ while true
     m_noFocus=$(grep 'Something is not right! Pogo is not in focus. Killing pogo and clearing junk' $monitor_log | wc -l)
     m_unknown=$(grep 'Something happened! Some kind of error' $monitor_log | wc -l)
 
+# version specific metrics
+if [ $android_version -ge 9 ]; then
+    cpuSys=$(top -n 1 | grep %sys | awk 'NR == 1 {sub(/%sys/, "", $4); print $4}')
+    cpuUser=$(top -n 1 | grep %user | awk 'NR == 1 {sub(/%user/, "", $2); print $2}')
+    cpuPogoPct=$(dumpsys cpuinfo | grep 'com.nianticlabs.pokemongo' | awk '{print substr($1, 1, length($1)-1)}')
+    cpuApct=$(dumpsys cpuinfo | grep 'com.pokemod.atlas' | awk 'NR == 1 {sub(/%/, "", $1); print $1}')
+    #Still not sure about these
```

Comment:
Has anyone tested this yet? Maybe need to ask in PoGoRom Dev server.

Reply:
tested by me, feel free to test.

Reply:
Working for me too on A9 S905W.